Modules/IO/Meta/test/itkLargeMetaImageWriteReadTest.cxx

```diff
@@ -143,9 +133,8 @@ ActualTest(std::string filename, typename TImageType::SizeType size)
       std::cerr << "Read Image pixel value " << ritr.Get() << std::endl;
       return EXIT_FAILURE;
     }
-
-    ++pixelValue;
-    ++ritr;
+    // NOTE: pixelValue for unsigned short will wrap around with large images.
+    pixelValue = (pixelValue != itk::NumericTraits<PixelType>::max()) ? ++pixelValue : 0;
```

Comment:
The same GCC warning as at the line 80 before:

https://open.cdash.org/viewBuildError.php?type=1&buildid=8544205